apps/wpcom-block-editor/src/calypso/features/gutenboarding-editor-overrides.js

```diff
@@ -54,3 +58,65 @@ function updateSettingsBar() {
 		settingsBar.prepend( saveButton );
 	} );
 }
+
+function showFirstTimeNotice() {
+	const getPreviewButton = () => document.querySelector( '.editor-post-preview' );
+
+	const awaitPreviewButton = setInterval( () => {
+		const previewButton = getPreviewButton();
+		if ( ! previewButton ) {
+			return;
+		}
+		clearInterval( awaitPreviewButton );
+
+		// Create preview link
+		// Reuse the href from existing Preview button in the header
+		// and trigger the click event handler (registered through jQuery)
+		// as seen in handlePreview() in iframe-bridge-server.js.
+		const previewLink = (
+			<a
+				href={ previewButton.href }
+				onClick={ ( e ) => {
+					e.preventDefault();
+					// Need to requery this element because the PreviewButton
+					// gets recreated when window is resized hence referencing
+					// an old destroyed button wouldn't work.
+					getPreviewButton().click();
```

Comment:
I wonder if it's better to hook into this function vs clicking the button via DOM?

https://github.com/Automattic/wp-calypso/blob/d367491eb6feb366e8f35e54d9023fbb4282e9ce/apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js#L462-L537

Reply:
That said, this message is quite temporary so I wouldn't be overly worried how it's done. :-)

Reply:
This function registers the event handler, the event handler itself is inside the function. So will need to take the function out and put it on the root level, give it a function name to use it in this file and export it to use it in the other file.

It's certainly possible, and I think it's good to do so too. This, together with using `href="javascript: void(0)"` for https://github.com/Automattic/wp-calypso/pull/41661/files#diff-2fb31dde2e9790f5f9a50030506f2c16R78, we can decouple the notice's dependency on the preview button and no longer need to `awaitPreviewButton`.

Reply:
Nevermind, it turns out to be really weird code cleanliness wise. Sticking to `.click()`.